create-project-async/helper.py

```diff
@@ -0,0 +1,14 @@
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+import json
+
+def get_access_token(base_url, client_id, client_secret):
+    client = BackendApplicationClient(client_id=client_id)
+    oauth = OAuth2Session(client=client)
+    token = oauth.fetch_token(token_url=base_url + '/api/oauth/token', client_id=client_id,
```

Comment:
what happened if user put wrong client_id and client_secret? 

is the error message from terminal friendly?

Reply:
It will show Invalid client: client is invalid but with stack trace like bellow, should we filter it?
```
Traceback (most recent call last):
  File "/Users/dedy/Documents/github/datasaur-ai/api-client-example/create-project-async/get_job_status.py", line 26, in <module>
    access_token = get_access_token(BASE_URL, CLIENT_ID, CLIENT_SECRET)
  File "/Users/dedy/Documents/github/datasaur-ai/api-client-example/create-project-async/helper.py", line 8, in get_access_token
    token = oauth.fetch_token(token_url=base_url + '/api/oauth/token', client_id=client_id,
  File "/usr/local/lib/python3.9/site-packages/requests_oauthlib/oauth2_session.py", line 360, in fetch_token
    self._client.parse_request_body_response(r.text, scope=self.scope)
  File "/usr/local/lib/python3.9/site-packages/oauthlib/oauth2/rfc6749/clients/base.py", line 421, in parse_request_body_response
    self.token = parse_token_response(body, scope=scope)
  File "/usr/local/lib/python3.9/site-packages/oauthlib/oauth2/rfc6749/parameters.py", line 431, in parse_token_response
    validate_token_parameters(params)
  File "/usr/local/lib/python3.9/site-packages/oauthlib/oauth2/rfc6749/parameters.py", line 438, in validate_token_parameters
    raise_from_error(params.get('error'), params)
  File "/usr/local/lib/python3.9/site-packages/oauthlib/oauth2/rfc6749/errors.py", line 405, in raise_from_error
    raise cls(**kwargs)
oauthlib.oauth2.rfc6749.errors.InvalidClientError: (invalid_client) Invalid client: client is invalid
```